lib/util.js

```diff
@@ -172,6 +172,27 @@ util = {
 
                 return callback(null, value.toString());
             });
+    },
+
+    /**
+     * Helper function to test if a given string is an integer.
+     *
+     * @param {String} value - The string to test for.
+     * @returns {Boolean}
+     */
+    isInt: function (value) {
```

Comment:
Were these taken from csv-parse? If so, add a comment linking to the original code in this, as well as the function below.

Reply:
Yes, added the reference in the below function. Will add a proper description.